src/pages/partners/affiliate-ib/_deriv-affiliate-programme.js

```diff
@@ -186,8 +186,18 @@ const Turnover = [
     },
 ]
 const DerivAffiliateProgramme = () => {
+    useEffect(() => {
+        const anchor = window.location.hash.slice(1)
+        if (anchor) {
+            const el = document.getElementById(anchor)
+            const top = window.scrollY + el.getBoundingClientRect().top - 50
+            window.scrollTo({ top, behavior: 'smooth' })
+            history.replaceState({}, document.title, '.')
+        }
+    }, [])
+
     return (
-        <StyledSection>
+        <StyledSection id="deriv-ap">
```

Comment:
What is this id doing ?

Reply:
This id here is working as anchor, it needed for transition to this part of the page from affiliate form.